src/views/login/login.view.tsx

```diff
@@ -40,23 +40,29 @@ const Login: FC = () => {
       <Typography type="body1" className={classes.appName}>
         Bridge
       </Typography>
-      <Card className={classes.card}>
-        {selectedWallet === undefined ? (
-          <>
-            <Typography type="h1" className={classes.cardHeader}>
-              Connect a wallet
-            </Typography>
-            <WalletList onSelectWallet={onConnectProvider} />
-          </>
-        ) : (
-          <>
-            <Typography type="h1" className={`${classes.cardHeader} ${classes.cardHeaderCentered}`}>
-              Connecting to
-            </Typography>
-            <AccountLoader selectedWallet={selectedWallet} />
-          </>
-        )}
-      </Card>
+      <div className={classes.cardWrap}>
+        <Card className={classes.card}>
+          {selectedWallet === undefined ? (
+            <>
+              <Typography type="h1" className={classes.cardHeader}>
+                Connect a wallet
+              </Typography>
+              <WalletList onSelectWallet={onConnectProvider} />
+            </>
+          ) : (
+            <>
+              <Typography
+                type="h1"
+                className={`${classes.cardHeader} ${classes.cardHeaderCentered}`}
+              >
+                Connecting to
+              </Typography>
+              <AccountLoader selectedWallet={selectedWallet} />
+            </>
+          )}
+        </Card>
+        {account.status === "failed" && <Error error={account.error} />}
```

Comment:
I think that this error should disappear when you click again in a wallet to login. I think that this should work:

```
{selectedWallet === undefined && account.status === "failed" && <Error error={account.error} />}
```

Reply:
It should be already disappearing since `account` goes back to `{status: "loading"}`